packages/webhook-utils/src/update-webhooks.ts

```diff
@@ -0,0 +1,18 @@
+import { Client } from "urql";
+import { getAppWebhooks } from "./operations/get-app-details-and-webhooks-data";
+
+interface updateWebhooksArgs {
+  client: Client;
+}
+
+export const updateWebhooks = async ({ client }: updateWebhooksArgs) => {
```

Comment:
shouldn't it receive webhooks in arguments? (WebhookManifests?)

Reply:
Updated the function to your suggestion 👍